### DISCUSSION TITLE
Add preview for svg and video files

### DISCUSSION THREAD

**Discussion by lotarbo** [awcodes/filament-curator#126] [Ideas]
Hello, i think it would be nice to see content in preview, not a icon file
<img width="340" alt="image" src="https://user-images.githubusercontent.com/1766890/224390642-0b2b42f8-3fdc-4066-a254-98a746e772c1.png">
<img width="333" alt="image" src="https://user-images.githubusercontent.com/1766890/224390680-c77c7f83-70f7-4f60-806b-6900750d6d0b.png">
 
In media item preview it already works
<img width="846" alt="image" src="https://user-images.githubusercontent.com/1766890/224390796-c3739348-9775-46be-9385-7537d8a001c0.png">

**Comment by awcodes** [comment]
I'm ok with supporting svg previews. But what would you recommend for video previews. There's no video thumbnails and I'm not comfortable  rendering actual videos in an index.
